OmiseSDK/Capability.swift

```diff
@@ -345,22 +356,29 @@ private let creditCardBackendTypeValue = "card"
 extension Capability.Backend {
     fileprivate enum BackendType: Codable, Hashable {
         case card
-        case source(OMSSourceTypeValue)
+        case source(SourceTypeValue)
+        case unknown(sourceType: String)
 
         init(from decoder: Decoder) throws {
             let container = try decoder.singleValueContainer()
             switch try container.decode(String.self) {
             case creditCardBackendTypeValue:
                 self = .card
             case let value:
-                self = .source(OMSSourceTypeValue(value))
+                if let sourceType = SourceTypeValue(value) {
+                    self = .source(sourceType)
+                } else {
+                    self = .unknown(sourceType: value)
+                }
```

Comment:
WDYT about using ternary operator?

```suggestion
                let sourceType = SourceTypeValue(value)
                 self = sourceType ? .source(sourceType) : unknown(sourceType: value)
```